### ISSUE TITLE
common/utils.go:249:34: undefined: common.HeightRPCCompatibility

### ISSUE THREAD

**Issue by mishra-keshav** [thetatoken/theta-eth-rpc-adaptor#29]
ubuntu@ip-172-31-6-115:/usr/local/go/src/github.com/thetatoken/theta-eth-rpc-adaptor$ make install
go install ./cmd/...
# github.com/thetatoken/theta-eth-rpc-adaptor/common
common/utils.go:249:34: undefined: common.HeightRPCCompatibility
make: *** [Makefile:24: release] Error 2

**Comment by jieyilong** [comment]
Thanks for the question. Did you checkout the `theta` code base following the instructions in the link below?

https://docs.thetatoken.org/docs/setup-the-eth-rpc-adaptor-for-the-theta-mainnet#21-install-the-adaptor

**Comment by mishra-keshav** [comment]
Yes @jieyilong , I have followed the same documentation which you have provided.
And also the code directories is as same as you have suggested but then also i am getting same error.

To compile the code of rpc adapter, make install command will be executed in which directory ?
thetatoken/theta
or
thetatoken/theta-eth-rpc-adaptor
